### PR TITLE
[rand.req.eng] Omit superfluous dollar-math wrapping inside \bigoh.

### PR DESCRIPTION
**Pull request by Eelis**
Compare: [line 2070](https://github.com/cplusplus/draft/blob/master/source/numerics.tex#L2070)

### CODE REVIEW

File: source/numerics.tex [hunk 1]
```diff
@@ -1912,21 +1912,21 @@
     with the same initial state
     as all other default-constructed engines
     of type \tcode{E}.
-  & \bigoh{$\mbox{size of state}$}
```
Comment (tkoeppe) [inline source/numerics.tex]: Why `\mbox` and not `\text`?

Comment (Eelis) [inline source/numerics.tex]: Beats me, I tried reading https://tex.stackexchange.com/questions/70632/difference-between-various-methods-for-producing-text-in-math-mode but it gave me a headache :)

**Comment by tkoeppe** [comment]
Let's leave this for now. I can revise the section systematically if and when I get to it.

**Comment by Eelis** [comment]
I'm curious, is that due to a policy that an issue in a line of code may only be fixed if all other potential issues on the same line are fixed at the same time?

**Comment by tkoeppe** [comment]
Just triaging the deluge of pull requests. I would make this change differently and probably will at some point, so I figured time is better spend on other things.

**Pull request closed without merging**